pkgs/development/compilers/bs-platform/default.nix

```
@@ -4,14 +4,14 @@ let
 in
 (build-bs-platform {
   inherit stdenv runCommand fetchFromGitHub ninja nodejs python3;
-  version = "7.0.1";
+  version = "7.1.0";
```

revert version bump:
```suggestion
  version = "7.0.1";
```